docker-compose-generator/docker-fragments/opt-add-sphinxrelay.yml

```diff
@@ -0,0 +1,39 @@
+version: "3"
+services:
+  btcpayserver:
+    environment:
+      BTCPAY_EXTERNALSERVICES: "Sphinx Relay:${BTCPAY_PROTOCOL:-https}://${BTCPAY_HOST}/sphinxrelay/app;"
+    volumes:
+      - "sphinxrelay_datadir:/etc/sphinxrelay_datadir"
+  lnd_bitcoin:
+    expose:
+      - "10009"
+  sphinxrelay:
+    image: sphinxlightning/sphinx-relay:v2.0.12
+    user: "0:0"
+    restart: unless-stopped
+    expose:
+      - "3300"
+    volumes:
+      - "sphinxrelay_datadir:/relay/.lnd"
+      - "lnd_bitcoin_datadir:/relay/lnd:ro"
```

Comment:
I wonder if you could share only the macaroon and not all the folder.

Reply:
Tried to search google, does not seem possible

Reply:
I've tried mounting single files and also didn't find anything. That's also why I needed the PR in Sphinx Relay itself – my initial idea was mounting the macaroons to the locations Sphinx expects, but that didn't work.